src/main/java/com/minecolonies/coremod/entity/ai/util/Structure.java

```diff
@@ -296,7 +297,7 @@ private static StructureWrapper loadStructure(
      */
     public Structure(final World targetWorld, final StructureWrapper structure, final Stage stageProgress)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this constructor to comply with Java Code Conventions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1213)